en/flight_modes/offboard.md

```diff
@@ -2,12 +2,17 @@
 
 [<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
 
-The vehicle obeys position, velocity, acceleration, attitude, attitude rates or thrust/torque setpoints provided by some source that is external to the flight stack, such as a companion computer.
-The setpoints may be provided using MAVLink (or a MAVLink API such as [MAVSDK](https://mavsdk.mavlink.io/)) or by [ROS 2](../ros/ros2.md).
+The _Offboard_ mode allows PX4 to be controlled by some source that is external to the flight stack, such as a companion computer.
```

Comment:
Note to self. Intent is to push the ROS 2 specifics and MAVLink specifics as far as possible into those instructions. The MAVLink part more or less done but the ROS 2 not yet done, as is pending response to https://github.com/PX4/PX4-user_guide/pull/3031#discussion_r1480614076

When that is done probably merge the content in the top of page and description into the description (as far as possible) as there is duplication now.